### ISSUE TITLE
Placing one pipe crashing the Game

### ISSUE THREAD

**Issue by Lonewolf1893** [refinedmods/refinedpipes#105]
### Describe the bug

Hello. i have placed one Item Pipe level 3 and the Game crashed to desktop with this error: 
The game crashed whilst tesselating block in world
Error: java.lang.NullPointerException: Cannot invoke "net.minecraft.client.resources.model.BakedModel.m_7541_()" because "this.core" is null

The Pipes are invisible and crashing after placing it on the ground

I have tried another pipes and i get everytime the same error

### How can we reproduce this bug or crash?

1.Placing a Pipe on the ground

### What Minecraft version is this happening on?

Minecraft 1.18.1

### What Forge version is this happening on?

39.0.45

### What Refined Pipes version is this happening on?

0.6.0 for 1.181.1

### Relevant log output

```shell
[21Jan.2022 12:05:36.070] [main/INFO] [cpw.mods.modlauncher.Launcher/MODLAUNCHER]: ModLauncher running: args [--username, Fenrir_Grauwolf, --version, 1.18.1-forge-39.0.45, --gameDir, C:\Users\weber\AppData\Roaming\.minecraft, --assetsDir, C:\Users\weber\AppData\Roaming\.minecraft\assets, --assetIndex, 1.18, --uuid, 9e96e044154c495ea799697bd6e73647, --accessToken, ????????, --clientId, MzhlZDg5OTEtOWQxMC00NDYwLTk4MGMtMjcyODQxZmM5NTU2, --xuid, 2535441389750483, --userType, msa, --versionType, release, --launchTarget, forgeclient, --fml.forgeVersion, 39.0.45, --fml.mcVersion, 1.18.1, --fml.forgeGroup, net.minecraftforge, --fml.mcpVersion, 20211210.034407]
[21Jan.2022 12:05:36.078] [main/INFO] [cpw.mods.modlauncher.Launcher/MODLAUNCHER]: ModLauncher 9.1.0+9.1.0+main.6690ee51 starting: java version 17.0.1 by Microsoft
[21Jan.2022 12:05:36.236] [main/INFO] [mixin/]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=union:/C:/Users/weber/AppData/Roaming/.minecraft/libraries/org/spongepowered/mixin/0.8.5/mixin-0.8.5.jar%2319!/ Service=ModLauncher Env=CLIENT
[21Jan.2022 12:05:38.300] [main/INFO] [cpw.mods.modlauncher.LaunchServiceHandler/MODLAUNCHER]: Launching target 'forgeclient' with arguments [--version, 1.18.1-forge-39.0.45, --gameDir, C:\Users\weber\AppData\Roaming\.minecraft, --assetsDir, C:\Users\weber\AppData\Roaming\.minecraft\assets, --uuid, 9e96e044154c495ea799697bd6e73647, --username, Fenrir_Grauwolf, --assetIndex, 1.18, --accessToken, ????????, --clientId, MzhlZDg5OTEtOWQxMC00NDYwLTk4MGMtMjcyODQxZmM5NTU2, --xuid, 2535441389750483, --userType, msa, --versionType, release]
[12:05:47] [Render thread/WARN]: Assets URL 'union:/C:/Users/weber/AppData/Roaming/.minecraft/libraries/net/minecraft/client/1.18.1-20211210.034407/client-1.18.1-20211210.034407-srg.jar%2353!/assets/.mcassetsroot' uses unexpected schema
[12:05:47] [Render thread/WARN]: Assets URL 'union:/C:/Users/weber/AppData/Roaming/.minecraft/libraries/net/minecraft/client/1.18.1-20211210.034407/client-1.18.1-20211210.034407-srg.jar%2353!/data/.mcassetsroot' uses unexpected schema
[21Jan.2022 12:05:47.613] [Render thread/INFO] [com.mojang.authlib.yggdrasil.YggdrasilAuthenticationService/]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[12:05:48] [Render thread/INFO]: Setting user: Fenrir_Grauwolf
[12:05:48] [Render thread/INFO]: Backend library: LWJGL version 3.2.2 SNAPSHOT
[12:05:50] [modloading-worker-0/INFO]: Forge mod loading, version 39.0.45, for MC 1.18.1 with MCP 20211210.034407
[12:05:50] [modloading-worker-0/INFO]: MinecraftForge v39.0.45 Initialized
[21Jan.2022 12:06:00.705] [Render thread/INFO] [com.mojang.text2speech.NarratorWindows/]: Narrator library for x64 successfully loaded
[12:06:01] [Render thread/INFO]: Reloading ResourceManager: Default, Mod Resources
[21Jan.2022 12:06:01.398] [Forge Version Check/INFO] [net.minecraftforge.fml.VersionChecker/]: [forge] Starting version check at https://files.minecraftforge.net/net/minecraftforge/forge/promotions_slim.json
[21Jan.2022 12:06:02.605] [Forge Version Check/INFO] [net.minecraftforge.fml.VersionChecker/]: [forge] Found status: BETA_OUTDATED Current: 39.0.45 Target: 39.0.55
[12:06:13] [Worker-Main-5/WARN]: Unable to resolve texture reference: #missing in refinedpipes:item/advanced_fluid_pipe
[12:06:13] [Worker-Main-5/WARN]: Unable to resolve texture reference: #missing in refinedpipes:item/advanced_item_pipe
[12:06:13] [Worker-Main-5/WARN]: Unable to resolve texture reference: #missing in refinedpipes:item/improved_item_pipe
[12:06:13] [Worker-Main-5/WARN]: Unable to resolve texture reference: #missing in refinedpipes:item/ultimate_fluid_pipe
[12:06:13] [Worker-Main-5/WARN]: Unable to resolve texture reference: #missing in refinedpipes:item/elite_energy_pipe
[12:06:13] [Worker-Main-5/WARN]: Unable to resolve texture reference: #missing in refinedpipes:item/advanced_energy_pipe
[12:06:13] [Worker-Main-5/WARN]: Unable to resolve texture reference: #missing in refinedpipes:item/basic_item_pipe
[12:06:13] [Worker-Main-5/WARN]: Unable to resolve texture reference: #missing in refinedpipes:item/improved_energy_pipe
[12:06:13] [Worker-Main-5/WARN]: Unable to resolve texture reference: #missing in refinedpipes:item/basic_fluid_pipe
[12:06:13] [Worker-Main-5/WARN]: Unable to resolve texture reference: #missing in refinedpipes:item/basic_energy_pipe
[12:06:13] [Worker-Main-5/WARN]: Unable to resolve texture reference: #missing in refinedpipes:item/ultimate_energy_pipe
[12:06:13] [Worker-Main-5/WARN]: Unable to resolve texture reference: #missing in refinedpipes:item/improved_fluid_pipe
[12:06:13] [Worker-Main-5/WARN]: Unable to resolve texture reference: #missing in refinedpipes:item/elite_fluid_pipe
[12:06:29] [Render thread/INFO]: OpenAL initialized on device OpenAL Soft on Lautsprecher (Realtek(R) Audio)
[12:06:29] [Render thread/INFO]: Sound engine started
[12:06:29] [Render thread/INFO]: Created: 1024x512x4 minecraft:textures/atlas/blocks.png-atlas
[12:06:29] [Render thread/INFO]: Created: 256x128x4 minecraft:textures/atlas/signs.png-atlas
[12:06:29] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/banner_patterns.png-atlas
[12:06:29] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/shield_patterns.png-atlas
[12:06:29] [Render thread/INFO]: Created: 256x256x4 minecraft:textures/atlas/chest.png-atlas
[12:06:29] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/beds.png-atlas
[12:06:29] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/shulker_boxes.png-atlas
[12:06:33] [Render thread/INFO]: Created: 256x256x0 minecraft:textures/atlas/particles.png-atlas
[12:06:33] [Render thread/INFO]: Created: 256x256x0 minecraft:textures/atlas/paintings.png-atlas
[12:06:33] [Render thread/INFO]: Created: 128x128x0 minecraft:textures/atlas/mob_effects.png-atlas
[12:06:39] [Render thread/INFO]: Injecting existing registry data into this CLIENT instance
[12:06:44] [Render thread/WARN]: Ambiguity between arguments [teleport, location] and [teleport, destination] with inputs: [0.1 -0.5 .9, 0 0 0]
[12:06:44] [Render thread/WARN]: Ambiguity between arguments [teleport, location] and [teleport, targets] with inputs: [0.1 -0.5 .9, 0 0 0]
[12:06:44] [Render thread/WARN]: Ambiguity between arguments [teleport, destination] and [teleport, targets] with inputs: [Player, 0123, @e, dd12be42-52a9-4a91-a8a1-11c01849e498]
[12:06:44] [Render thread/WARN]: Ambiguity between arguments [teleport, targets] and [teleport, destination] with inputs: [Player, 0123, dd12be42-52a9-4a91-a8a1-11c01849e498]
[12:06:44] [Render thread/WARN]: Ambiguity between arguments [teleport, targets, location] and [teleport, targets, destination] with inputs: [0.1 -0.5 .9, 0 0 0]
[12:06:44] [Render thread/INFO]: Reloading ResourceManager: Default, forge-1.18.1-39.0.45-universal.jar, refinedpipes-0.6.0.jar
[12:06:48] [Render thread/INFO]: Loaded 7 recipes
[12:06:50] [Render thread/INFO]: Loaded 1141 advancements
[21Jan.2022 12:06:54.341] [Render thread/INFO] [com.mojang.authlib.yggdrasil.YggdrasilAuthenticationService/]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[12:06:54] [Server thread/INFO]: Starting integrated minecraft server version 1.18.1
[12:06:54] [Server thread/INFO]: Generating keypair
[12:06:54] [Server thread/INFO]: Preparing start region for dimension minecraft:overworld
[12:06:56] [Render thread/INFO]: Einstiegsbereich wird vorbereitet: 0 %
[12:06:59] [Render thread/INFO]: Einstiegsbereich wird vorbereitet: 0 %
[12:06:59] [Render thread/INFO]: Einstiegsbereich wird vorbereitet: 0 %
[12:06:59] [Render thread/INFO]: Einstiegsbereich wird vorbereitet: 0 %
[12:06:59] [Render thread/INFO]: Einstiegsbereich wird vorbereitet: 0 %
[12:06:59] [Render thread/INFO]: Einstiegsbereich wird vorbereitet: 0 %
[12:06:59] [Render thread/INFO]: Einstiegsbereich wird vorbereitet: 0 %
[12:06:59] [Render thread/INFO]: Einstiegsbereich wird vorbereitet: 0 %
[12:06:59] [Render thread/INFO]: Einstiegsbereich wird vorbereitet: 0 %
[12:06:59] [Render thread/INFO]: Einstiegsbereich wird vorbereitet: 0 %
[12:06:59] [Render thread/INFO]: Einstiegsbereich wird vorbereitet: 4 %
[12:07:00] [Render thread/INFO]: Einstiegsbereich wird vorbereitet: 14 %
[12:07:00] [Render thread/INFO]: Einstiegsbereich wird vorbereitet: 20 %
[12:07:01] [Render thread/INFO]: Einstiegsbereich wird vorbereitet: 44 %
[12:07:01] [Render thread/INFO]: Einstiegsbereich wird vorbereitet: 56 %
[12:07:02] [Render thread/INFO]: Einstiegsbereich wird vorbereitet: 69 %
[12:07:02] [Render thread/INFO]: Einstiegsbereich wird vorbereitet: 79 %
[12:07:03] [Render thread/INFO]: Einstiegsbereich wird vorbereitet: 82 %
[12:07:03] [Render thread/INFO]: Einstiegsbereich wird vorbereitet: 83 %
[12:07:04] [Render thread/INFO]: Einstiegsbereich wird vorbereitet: 84 %
[12:07:04] [Render thread/INFO]: Einstiegsbereich wird vorbereitet: 92 %
[12:07:05] [Render thread/INFO]: Einstiegsbereich wird vorbereitet: 96 %
[12:07:05] [Render thread/INFO]: Einstiegsbereich wird vorbereitet: 98 %
[12:07:05] [Server thread/INFO]: Successfully initialized permission handler forge:default_handler
[12:07:05] [Render thread/INFO]: Time elapsed: 11128 ms
[12:07:06] [Server thread/INFO]: Changing view distance to 12, from 10
[12:07:06] [Server thread/INFO]: Changing simulation distance to 12, from 0
[12:07:07] [Netty Local Client IO #0/INFO]: Connected to a modded server.
[12:07:07] [Server thread/INFO]: Using new advancement loading for net.minecraft.server.PlayerAdvancements@33581dc2
[12:07:07] [Server thread/INFO]: Fenrir_Grauwolf[local:E:351e48a7] logged in with entity id 193 at (668.5, 66.0, 422.5)
[12:07:07] [Server thread/INFO]: Fenrir_Grauwolf hat das Spiel betreten
[12:07:08] [Render thread/INFO]: Loaded 0 advancements
```

**Comment by Pbone3** [comment]
I've had this issue on the same versions as well; it affects every pipe, not just the one you've tried.

**Comment by raoulvdberge** [comment]
Fixed by #104
